File: app/src/main/java/com/th/forge/taxiorders/api/RoxieApiService.java
package com.th.forge.taxiorders.api;

import com.th.forge.taxiorders.Entity.Order;

import java.util.ArrayList;


import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface RoxieApiService {
    @GET("orders.json")
    Call<ArrayList<Order>> getOrders();

    @GET("images/{imageName}")
    Call<ResponseBody> getImage(@Path("imageName") String imageName);
}

File: app/src/main/java/com/th/forge/taxiorders/api/RoxieApiService.kt
package com.th.forge.taxiorders.api

import com.th.forge.taxiorders.entity.Order

import java.util.ArrayList


import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface RoxieApiService {
    @get:GET("orders.json")
    val orders: Call<ArrayList<Order>>

    @GET("images/{imageName}")
    fun getImage(@Path("imageName") imageName: String): Call<ResponseBody>
}
